test/kata1.fizzBuzz.test.js

```diff
@@ -1,11 +1,18 @@
 const { fizzBuzz } = require("../src");
 
 describe("fizzBuzz", () => {
-  test("returns Fizz when passed a multiple of 3", () => {});
+  test("returns Fizz when passed a multiple of 3", () => {
+    expect(fizzBuzz(3)).toEqual('Fizz');
```

Comment:
In the require, describe and test blocks you are using double quotation marks "", and for better consistency I would suggest to stick with the "" throughout the app. 
When it comes to strings you are better off using doubles "" everywhere as when you would say "My son's name is Tom" works with double quotations, but it would not work with singles: 'My son's name is Tom', as the apostrophe would close the string and throw an error.